scripts/latest

```diff
@@ -0,0 +1 @@
+ghcr.io/sifchain/sifnode/ui-stack:0dfa9f7eab92d342bf1488296fce34bdf50c8918
```

Comment:
Seems strange to have a file in the toplevel scripts directory just called "latest"